### ISSUE TITLE
In IE, accessing a video that's on another domain gets me "Access Denied" error.

### ISSUE THREAD

**Issue by david-goliath** [nyroDev/nyroModal#27]
In this case, it was a Vimeo video.

Tried it in IE 7 and 8.

**Comment by nyroDev** [comment]
I can't reproduce this bug.
Could you explain more on how to reproduce it, or better, send a link?
BTW, on which version of IE are you testing?

**Comment by skashi** [comment]
Video's don't show up at all in IE. 

I've tried in all major browsers (Firefox, Chrome, Opera, Safari) and they all display the video just fine but in IE8 and IE7 it just displays the loading gif.

This is happening on the demo (both youtube links):
http://nyromodal.nyrodev.com/

**Comment by skashi** [comment]
Ok I've narrowed down the issue:

If I try clicking on the youtube links on your demo on my laptop running Windows XP 32bit the videos show up fine in IE8.

However, doing the same thing on Windows 7 64bit it just displays the loading gif.

I picked this up because I noticed when I tried out this site: http://client.baigent.net/playertest/nyrotube.html (from issue58) and clicked on the image it was telling me that I needed to upgrade my Adobe Flash player. I clicked on that link and it said: "Flash Player 10.1 is not currently available for your 64-bit web browser".

I am not sure whether these two issues are the same. If so, is this an issue that be resolved without having to wait for Adobe to update their 64bit Flash Player 10.1 version?
